hub/management/commands/import_council_emergency_declaration.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
+from mysoc_dataset import get_dataset_url
+
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+declare_map = {
+    "Y": "Yes",
+    "N": "No",
+}
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss_code"
+    message = "Importing council climate emergency declarations"
+    uses_gss = True
+    do_not_convert = True
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "label": "Climate Emergency Declaration",
```

Comment:
I’d word this as “Has declared a climate emergency”… or maybe “Council has declared a climate emergency” to make the scope clearer when you’re using this dataset on the `/explore` page?